pkg/metrics/metrics.scss

```diff
@@ -284,23 +289,41 @@
         grid-row: 0;
         grid-column: 1 / -1;
         background: #eee;
+        margin-left: var(--pf-global--spacer--md);
     }
 
     &-events {
         grid-column: events;
         grid-row: calc(var(--metrics-minutes) - var(--metrics-minute));
-        display: grid;
-        grid-template-columns: [info] 1fr [time] 1fr;
-        align-items: baseline;
 
-        dt {
-            grid-column: time;
-            font-size: var(--pf-global--FontSize--sm);
-            grid-row: 1;
+        .description {
+            display: grid;
+            grid-template-columns: [info] 1fr [time] 1fr;
+            align-items: baseline;
```

Comment:
align items and justify items are not causing any visual difference neither in desktop neither in mobile sizes when removed. 
I can see that it was before there, but now it seems to apply on another element.